contrib/vspec2ttl/vspec2ttl.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python3
+
+# (C) 2021 BMW Group - All rights reserved.
+# AUTHOR: Daniel Wilms BMW Group;
+
+#
+#
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+#
+# Convert vspec file to TTL
+#
+import os
+import sys
+#Add path to main py vspec  parser
```

Comment:
\# (space)
and what is main py vspec <space> parser?   I suppose you mean `vspec.py`.

... and I wonder if there is a better way to do this for the contrib tools.  It seems a bit of a hack to have the program depend on where it is located in the tree.

Reply:
For some other tools that has been moved to contrib we solved it in the Makefile by setting PYTHONPATH before calling the script.


```
c:
	(cd ${TOOLSDIR}/contrib/vspec2c/; make )
	PYTHONPATH=${TOOLSDIR} ${TOOLSDIR}/contrib/vspec2c.py -i:./spec/VehicleSignalSpecification.id -I ./spec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).h vss_rel_$$(cat VERSION)_macro.h
```

Reply:
I think putting it in the python file is the correct way to do this, otherwise the tool is not usable without secret knowledge of some ENV variables that need to be set. As it is expected you check out the whole tools-repo it does not depend on where you locate it.

But that is just an opinion, I am fine either way 👍

Reply:
I'd vote for the python file as well.